agent/util-scripts/pbench-register-tool

```diff
@@ -132,7 +132,8 @@ function usage() {
 	#                     1         2         3         4         5         6         7         8
 	# (no tab)   12345678901234567890123456789012345678901234567890123456789012345678901234567890
 	printf -- "\nFor a list of tool specific options, run:\n"
-	printf -- "\t${pbench_bin}/tool-scripts/<tool-name> --help\n"
+	printf -- "\t${pbench_bin}/tool-scripts/<tool-name> --help\n\n"
+
```

Comment:
It's interesting that the blank line following the `printf` mimics the added spacing in the output, but it's not necessary. 😆